mod-source-record-storage-server/src/main/java/org/folio/services/RecordServiceImpl.java

```diff
@@ -250,6 +275,50 @@ public Future<Void> updateRecordsState(String matchedId, RecordState state, Reco
     return recordDao.updateRecordsState(matchedId, state, recordType, tenantId);
   }
 
+  private Future<Record> setMatchedIdForRecord(Record record, String tenantId) {
```

Comment:
please add some comments to the body of this method.

Reply:
done, thanks